service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterMapper.java

```diff
@@ -53,6 +53,10 @@ public String mapEncounterToEhrComposition(Encounter encounter) {
         LOGGER.debug("Generating ehrComposition for Encounter {}", encounter.getId());
         String components = encounterComponentsMapper.mapComponents(encounter);
 
+        if (StringUtils.isBlank(components)) {
+            return StringUtils.EMPTY;
```

Comment:
Is it possible to log a message out stating which resource is creating an empty string? provide the ID etc

Reply:
Addressed